### ISSUE TITLE
NTR: Scalp swelling, diffuse (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8016]
New term request Scalp swelling, diffuse (seidel_190211162202): Edema; swollen; head; skull;  UMLS: 2219601

**Comment by pnrobinson** [comment]
@MickeySegal
what is the intended use case?

**Comment by MickeySegal** [comment]
Tinea capitis
Non-accidental trauma

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000959
- label: Scalp edema
- parents: [Abnormal scalp morphology (HP:0001965)](https://hpo.jax.org/app/browse/term/HP:0001965) | [Edema (HP:0000969)](https://hpo.jax.org/app/browse/term/HP:0000969)
- pmid: 
- def: A palpable swelling in the scalp caused by an increase in interstitial fluid volume (excess fluid).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8016
- synonyms:
